src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.csproj

```diff
@@ -12,6 +12,8 @@
   </ItemGroup>
```

Comment:
Well to be correct it will be netstandard1.7 after @joperezr merge but as is I think it should be netstandard1.5